src/main/java/com/synopsys/arc/jenkinsci/plugins/jobrestrictions/util/GroupSelector.java

```diff
@@ -0,0 +1,130 @@
+/*
```

Comment:
You can use <code>org.jenkinsci.plugins.jobrestrictions.util.*</code> package if you want.
I have to use the company's package during my work-related contributions, but in general a company-independent package is preferable.